pandas/core/groupby/generic.py

```diff
@@ -179,25 +179,25 @@ def _post_process_cython_aggregate(self, obj):
             obj = obj.swapaxes(0, 1)
         return obj
 
-    def aggregate(self, arg, *args, **kwargs):
+    def aggregate(self, func, *args, **kwargs):
```

Comment:
The reason for this change is that MyPy was complaining about the signature here differening from what's defined in `SelectionMixin`. This aligns the signatures and is arguably more readable.

Counter argument is that this is an API change, but this actually would match the docs

<img width="768" alt="image" src="https://user-images.githubusercontent.com/609873/56101095-81c92980-5ed5-11e9-86e3-fc213eda023d.png">

Reply:
this is ok, pls add a note in the whatsnew about this though